cli/src/serve.js

```diff
@@ -229,6 +229,11 @@ const initialize_servers = (ctor, opts) => {
           console.info(`Static files being served from ${opts.serve_static}`);
         }
         srv.on('request', fileServer(opts.serve_static));
+      } else {
```

Comment:
I could be wrong, but I think this makes it so that the server won't serve the horizon.js file